easybuild/easyconfigs/g/GROMACS/README.md

```diff
@@ -104,11 +108,20 @@ engines.
     of enabling support for the `cray-python` modules. Therefore the corresponding 
     EasyConfigs of GROMACS have also been replaced.**
 
+### GROMACS 2023.2 with AMD GPU support for CPE 22.12
+
+  * There are multiple easyconfig files for running GROMACS with AMD GPU acceleration (ROCm support via hipSYCL), that follows [installation guide](https://manual.gromacs.org/2023.2/install-guide/index.html#sycl-gpu-acceleration-for-amd-gpus).
+
+    * MPI, hipSYCL with [HeFFTe](https://manual.gromacs.org/2023.2/install-guide/index.html#using-heffte) enabled version for multi-node use with support for multiple GPUs (PME decomposition)
+    * MPI, hipSYCL version without HeFFTe enabled (no PME decomposition), recommended for a single GPU use, not recommended for multi-node use
+    * thread-MPI, hipSYCL with [VkFFT](https://manual.gromacs.org/2023.2/install-guide/index.html#using-vkfft) version for a single-node use and single GPU offloading
```

Comment:
> * MPI, hipSYCL version without HeFFTe enabled (no PME decomposition)
> * thread-MPI, hipSYCL with VkFFT

Both these versions use VkFFT, so it's confusing to refer to the same setting as "without HeFFTe enabled (no PME decomposition)" and "with VkFFT".

The recommendations are also confusing:

> * recommended for a single GPU use, not recommended for multi-node use
> * version for a single-node use and single GPU offloading

It reads as if "thread-MPI" version is recommended for single-node multi-GPU use, which is not a good recommendation, because thread-MPI version does not support direct GPU communications (even on a single node). So, thread-MPI version is only useful for single-GPU runs; in any other case, "real" MPI is recommended.

Reply:
I have made another attempt to make description clear. Should it just recommend HeFFTe version for a multi-node (and multi-GPU) use cases and VkFFT for a single-GPU runs? Is it more clear now? Thanks for patience.